fxa-rust-client/ffi/src/lib.rs

```diff
@@ -46,9 +49,8 @@ pub unsafe extern "C" fn fxa_free(fxa: *mut FirefoxAccount) {
     let _ = Box::from_raw(fxa);
 }
 
-#[no_mangle]
-pub unsafe extern "C" fn fxa_config_free(config: *mut FxAConfig) {
-    let _ = Box::from_raw(config);
+pub unsafe extern "C" fn fxa_config_free(config: *mut Config) {
+    drop(Box::from_raw(config));
```

Comment:
Is there a difference here? My impression is `drop` is implemented as a function that takes ownership (via a move), so it would make no difference...

I don't disagree that this might be less unclear, but we probably should be consistent either way (and above you use the `let _ = ...` version).

Reply:
Yup I'm converting the rest of them in another PR. I switched to `drop` to make it clear that we're taking ownership and releasing the memory.